src/__tests__/canonicalFields.int.test.ts

```diff
@@ -1,14 +1,55 @@
 import request from 'supertest';
 import { app } from '../app';
+import { db } from '../database';
+import type { Result } from 'tinypg';
 
 const agent = request.agent(app);
 
 describe('/canonicalFields', () => {
   describe('/', () => {
-    it('should return HTTP Status Code 200 OK', async () => {
+    it('returns an empty array when no data is present', async () => {
       await agent
         .get('/canonicalFields')
-        .expect(200);
+        .expect(200, []);
+    });
+
+    it('returns all canonical fields present in the database', async () => {
+      await db.sql('fixtures.canonicalFields.returnsAllCanonicalFields.seed');
```

Comment:
It looks like this path is in the production code area, not the test code area:
```shell
$ find . -name "*seed*"
./src/database/queries/fixtures/canonicalFields/returnsAllCanonicalFields/seed.sql
./dist/database/queries/fixtures/canonicalFields/returnsAllCanonicalFields/seed.sql
```
I wonder if it is as easy to go into `src/test/database/fixtures` or something like that. I think `src/test/database/queries/fixtures/canonicalFields/returnsAllCanonicalFields` is too deep and long, but this comment is more a matter of style than substance.

Reply:
ok you've convinced me ;) -- I'm bailing on the tinypg-stored-procedure-whatever-you-wanna-call-it for test fixtures.  Too cheugy.